sos/plugins/libreswan.py

```diff
@@ -44,4 +54,17 @@ def setup(self):
             '/etc/ipsec.d/*.secrets'
         ])
 
+    def postproc(self):
+        # Remove any sensitive data.
+        # "ip xfrm state" output contains encryption or authentication private
+        # keys:
+        xfrm_state_regexp = r'(aead|auth|auth-trunc|enc)' \
+                            r'(\s.*\s)(0x[0-9a-f]+)'
+        self.do_cmd_output_sub("state", xfrm_state_regexp,
+                               r"\1\2********")
+
+        if self.get_option("ipsec-barf"):
```

Comment:
This test is not required as due to https://github.com/sosreport/sos/blob/master/sos/plugins/__init__.py#L494, a replacement will happen only if the command was collected. But it does not harm (and I think this code even saves few lines of code execution). So no problem with this.